### ISSUE TITLE
SEL calculation

### ISSUE THREAD

**Issue by e-sr** [python-acoustics/python-acoustics#193]
I expected the following relatioship between SEL and Leq:

$$ L_{eq} = SEL - 10 \log_{10}\frac{\delta T}{1s}  $$

or I am missing something?

We have presumibily an error in 

```
acoustics.standards.iso_tr_25417_2007.REFERENCE_EXPOSURE_LEVEL
```
and the exposure calculation has to account sampling rate.

**Comment by FRidh** [comment]
Hi Enzo,

Thanks for your report. You are indeed correct. The issue should be fixed in 8a3e4d5 . Can you confirm?

**Comment by e-sr** [comment]
Hi Frederik,
yes it's correct.
